functions/icons.zsh

```diff
@@ -86,6 +87,76 @@ case $POWERLEVEL9K_MODE in
   'awesome-fontconfig')
     # fontconfig with awesome-font required! See
     # https://github.com/gabrielelana/awesome-terminal-fonts
+    # if not defined, set recommended linux path
+    typeset -p "POWERLEVEL9K_FONTAWESOME_PATH" > /dev/null 2>&1  || POWERLEVEL9K_FONTAWESOME_PATH=~/.fonts
```

Comment:
Yep, unfortunately AFAIK on OSX doesn't work like that, and it's even more difficult with Sierra. Anyway, the mapping part is good even for OSX, the environment variable above probably doesn't work but it's harmless, I don't think it will "break" anything

@bhilburn what do you think it will break?